openmdao/solvers/linear/linear_runonce.py

```diff
@@ -13,6 +13,22 @@ class LinearRunOnce(LinearBlockGS):
 
     SOLVER = 'LN: RUNONCE'
 
+    def __init__(self, **kwargs):
+        """
+        Initialize all attributes.
+
+        Parameters
+        ----------
+        **kwargs : dict
+            Options dictionary.
+        """
+        super(LinearRunOnce, self).__init__(**kwargs)
+
+        # Remove unused options from base options.
+        self.options.undeclare("atol")
+        self.options.undeclare("rtol")
+        # self.options.undeclare("maxiter") # eventually want to undeclare this
```

Comment:
why is this "eventually" versus doing it now?

Reply:
Because it will take some rework on the solver call stack. Might have to rethink some things, and probably should be worked on by someone who is familiar with how the solvers work.

Reply:
I guess there's a story for that...  the comment on it's own, though, only serves to confuse (IMO)

Reply:
All gone! 😸